TestFlightCore/TestFlightCore/TestFlightCore.cs

```diff
@@ -40,9 +40,9 @@ public class TestFlightCore : PartModuleExtended, ITestFlightCore
         public float lastMET;
         [KSPField(isPersistant=true)]
         public bool initialized = false;
-        [KSPField(isPersistant=true)]
+        [KSPField]
         public float maxData = 0f;
-        [KSPField(isPersistant=true)]
```

Comment:
`maxData` is calculated from the reliability modules on the part, and if not persisted will have to be recalculated after load.  Which isn't necessarily an issue if preferred, but does take more than 0 time to complete. 

See: https://github.com/KSP-RO/TestFlight/blob/master/TestFlightCore/TestFlightCore/TestFlightCore.cs#L135